internal/consensus/state.go

```diff
@@ -137,7 +137,7 @@ type State struct {
 	done chan struct{}
 
 	// synchronous pubsub between consensus state and reactor.
-	// state only emits EventNewRoundStep and EventVote
+	// state only emits EventNewRoundStep, EventValidBlock, and EventVote
```

Comment:
Did this creep in from another branch?

Reply:
I changed it as I was investigating the event system since I noticed it was incorrect. happy to move into a different PR.

Reply:
That's fine, if you think it's relevant. The main reason I try to separate changes like this is so when I'm grepping log history I don't wind up getting a huge RFC along with a small file change (e.g., `git log internal/consensus/state.go` would show the merge commit from this PR but `git show` on it would be mostly the RFC).

All that is just by way of explanation, though—I don't feel strongly about it.